flask_monitoringdashboard/main.py

```diff
@@ -25,6 +26,16 @@ def endpoint():
         f()
         return 'Ok'
 
+    @app.route('/endpoint2')
```

Comment:
this does not seem to be related to postgres compatibility... or?

Reply:
Of course not:d Just added extra endpoints to test, which I then forgot to remove